go.mod

```diff
@@ -7,7 +7,9 @@ require (
 	cloud.google.com/go/storage v1.13.0 // indirect
 	github.com/agext/levenshtein v1.2.3 // indirect
 	github.com/apparentlymart/go-cidr v1.1.0 // indirect
-	github.com/aws/aws-sdk-go v1.37.10 // indirect
+	github.com/aws/aws-sdk-go v1.37.10
```

Comment:
is there a reason why we aren't using sdk v2 for aws?

Reply:
No idea, it was already there apparently 🤷‍♂️